setup_examples/ili9488_pico.py

```diff
@@ -0,0 +1,39 @@
+#
+# Setup for ILI9488 interfaced to Raspberry PI Pico 2 and Pico
+#
+# Released under the MIT License (MIT). See LICENSE.
+
+from micropython import const
+
+# Modify these Pin assignments to match your hardware.
+
+# Simple GPIO's
+LCD_DC        = 'GPIO21'  # Pin 27
```

Comment:
Pins on the Pico may be instantiated with numeric identifiers: this is more efficient than using strings. Please use numeric constants as per ESP32.